<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataPoint</class>
 <widget class="QWidget" name="dataPoint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>85</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>88</height>
      </size>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>0</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>结束时间</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>10</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="date">
       <date>
        <year>2021</year>
        <month>8</month>
        <day>19</day>
       </date>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd HH:mm:ss</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>10</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>生成报表</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="date">
       <date>
        <year>2021</year>
        <month>8</month>
        <day>7</day>
       </date>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd HH:mm:ss</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>巡检点位</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>开始时间</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>370</width>
         <height>0</height>
        </size>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>90</number>
       </attribute>
       <column>
        <property name="text">
         <string>巡检点</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>开始时间</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>钢棒温度</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>槽底温度</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(179, 249, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(179, 249, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
